### ISSUE TITLE
traffic_calming issue (Polish lang.)

### ISSUE THREAD

**Issue by ogor89** [osmandapp/OsmAnd#10520]
### Description
"traffic_calming=island
<https://wiki.openstreetmap.org/wiki/Pl:Tag:traffic_calming%3Disland>"
is reported as "traffic_calming=bump
<https://wiki.openstreetmap.org/wiki/Tag:traffic_calming%3Dbump>" in
Polish. It is directly in node No. 264286714
(https://www.openstreetmap.org/node/264286714).

### How to reproduce?
Go through node 264286714.

### Your Environment
OsmAnd Version: 3.8.4
Android/iOS version:7.0
Device model: tp-link neffos tp904a

**Maps used (online or offline):**   
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.

**Comment by sonora** [comment]
@ogor89 If you can propose a better Polish word, I would be happy to use it. Please note that it should cover all types of traffic calming, as we currently have only one type of event announcing traffic calmings, i.e. do not distinguish the many possible sub-types.

**Comment by sonora** [comment]
@vshcherb  a typical case for "Help Wanted", what does "Nice to have" mean to indicate here`?

**Comment by vshcherb** [comment]
Well definitely request is about support subtypes for traffic calming which I'm not sure we support or not or we could support. So I think it requires code change and code change requires adoption process but probably here it is not that difficult

**Comment by sonora** [comment]
A viable solution could be to find a word that works for all 'traffic calmings'. May work better or worse depending on the language, but certainly not impossible.

**Comment by JohnVeness** [comment]
See also #5605 (is this a duplication of that?)

**Comment by ticotaco72** [comment]
"uspokajacz ruchu" could probably be used, but I'm afraid similar issues could be brought up as in the czech example

**Comment by FreddieChopin** [comment]
A better word for "traffic_calming=island" would be "wysepka", as currently used word "próg zwalniający" is used only for a physical "bump" on the surface of the road, that you have to drive over (there are maybe a few types). You don't drive over an island.

Generally I think that "island" should not be announced at all, as this is so common that in some places you would have that every 20 meters. It is just not mapped that much <:

**Comment by ogor89** [comment]
> A better word for "traffic_calming=island" would be "wysepka", as currently used word "próg zwalniający" is used only for a physical "bump" on the surface of the road, that you have to drive over (there are maybe a few types). You don't drive over an island.
> 
> Generally I think that "island" should not be announced at all, as this is so common that in some places you would have that every 20 meters. It is just not mapped that much <:
I totally agree. Only some "traffic_calming" should be announced, like "bump".
